chef/cookbooks/monasca/recipes/master.rb

```diff
@@ -90,6 +90,13 @@
   keystone_admin_password: keystone_settings["admin_password"],
   database_grafana_password: node[:monasca][:master][:database_grafana_password],
 
+  notification_enable_email: node[:monasca][:master][:notification_enable_email],
+  smtp_host: node[:monasca][:master][:smtp_host],
+  smtp_port: node[:monasca][:master][:smtp_port],
+  smtp_user: node[:monasca][:master][:smtp_user],
+  smtp_password: node[:monasca][:master][:smtp_password],
+  smtp_from_address: node[:monasca][:master][:smtp_from_address],
```

Comment:
I don't know if this is correct. You are setting smtp to monasca master host. What if customer has smtp server and would like to configure monasca-notification to connect to this server??